### DISCUSSION TITLE
Ability to Add Fields to Meta/Secondary Panel

### DISCUSSION THREAD

**Discussion by RitterKnightCreative** [craftcms/cms#8663] [Ideas]
### Description
It's not often we need to do this but it'd be nice to be gain the ability to add fields to the meta/secondary panel of Craft. 

Plugins have the ability to do this but it'd be nice from the general authoring side as well.

The UI could be as simple as a checkbox in the new field creation area for "shows up in secondary area" or something to that affect.

A couple use cases:

* There's times where authors want to change the design/template of the site with a dropdown. You can use an entry type for this but that seems overkill, especially if all the fields stay the same.

* Oftentimes, we also use "hidden" pages, eg landing pages, that control whether the entry shows up in a navigation or not... or for SEO purposes, like setting noindex. This can also be a good use of a lightswitch in the meta area.

Just like anything in Craft, these sorts of things can get overused but the ability to put these sorts of controls in the secondary/meta area can also help declutter the main area, which is ideally used for content. 

Ideally, these sorts of controls are generally used for "set it and forget it" type controls, much like the slug and the enabled controls, where they are accessed rarely but are still necessary to the function of the entry.

**Comment by brandonkelly** [comment]
Technically this is a duplicate of #1009 but you’ve done a much better job making the case for it, so will close the other one instead.

**Reply by Oe-de-Cologne** [reply]
I support this suggestion. I mostly like Craft's UI, but I would prefer it to be more focused on creating content for editors (rather than dedicating such a large area permanently for metadata).
